mitodl/micromasters
# Added slug to FrequesntlyAskedQuestion

#### What are the relevant tickets?
Fix #1878
#### What's this PR do?
Added unique SlugField for FrequesnltyAskedQuestion in cms in order to reference questions in the url.
#### Where should the reviewer start?

#### How should this be manually tested?
Before you run the migration, make to have few existing faqs. Make sure to exit the editing mode for that page in wagtail (otherwise you will be seeing saved draft that will not show you any updates that took place in the models).
Then run the migration. Now when editing the `FrequesntlyAskedQuestion` objects you should see a third field `slug` that is populated with a generated slug from the question itself.

#### Any background context you want to provide?

#### Screenshots (if appropriate)
![screen shot 2016-12-21 at 4 41 27 pm](https://cloud.githubusercontent.com/assets/7574259/21406982/5fef2aee-c79c-11e6-8de1-33c47e3b343c.png)

#### What GIF best describes this PR or how it makes you feel?
